rero_mef/utils.py

```diff
@@ -200,7 +200,7 @@ def oai_process_records_from_dates(name, sickle, oai_item_iterator,
     last_run = None
     url, metadata_prefix, last_run, setspecs = get_info_by_oai_name(name)
 
-    request = sickle(url, iterator=oai_item_iterator)
+    request = sickle(url, iterator=oai_item_iterator, max_retries=10)
```

Comment:
can you put a small comment to explain why?

Reply:
corrected differently